source/EvoPlanet/evoplanet.client/src/app/types.ts

```diff
@@ -0,0 +1,23 @@
+//Waiting for backend to adapt the vector class
```

Comment:
These are TODO comments, which are actually one of the acceptable forms of code comments. You might want to annotate them with the TODO keyword (vscode can automatically find TODOs, and you can search for it easier in other editors; you just type TODO in the search bar; that way it's harder to forget them).